src/app/modules/analytics-config/services/powerbar-form-builder/powerbar-form-builder.service.ts

```diff
@@ -135,7 +135,7 @@ export class PowerbarConfigForm extends ConfigFormGroup {
           marker('Display the filter'),
           marker('powerbar display filter description')
         )
-      }).withTabName(marker('Data')),
+      }).withTabName(marker('Render')),
```

Comment:
`Data` instead ?

Reply:
it's the render part. It was a typo. We had two tabs of 'Data'

Reply:
Ok